src/EditorFeatures/Core.Wpf/NavigateTo/NavigateToItemProvider.Searcher.cs

```diff
@@ -84,19 +98,80 @@ private class Searcher
                 }
             }
 
-            private async Task SearchAsync(Project project)
+            private async Task SearchProjectsInPriorityOrder(IDocumentTrackingService docTrackingService)
+            {
+                var processedProjects = new HashSet<Project>();
+
+                var activeDocOpt = docTrackingService.GetActiveDocument(_solution);
```

Comment:
> isn't active doc already included in visible docs?

It may be... but it isn't doc'ed to be.  So i wrote with the assumption it might not be.  Also, even if it was, we still want to handle the active doc as *more* important than the visible docs.